src/stacktrace_constants.rs

```diff
@@ -222,3 +243,12 @@ pub const STACK_FRAME_FILEPATH_IGNORE_REGEXES: &[&str] = &[
     r".*libubsan\.so",
     r".*asan_with_fuzzer\.so",
 ];
+
+lazy_static::lazy_static! {
+    // Mutable variable for custom ignored function names
+    pub static ref STACK_FRAME_FUNCTION_IGNORE_REGEXES: RwLock<Vec<String>> = RwLock::new(
+        Vec::new());
+    // Mutable variable for custom ignored file paths
```

Comment:
Regular expressions for file paths to be ignored.

Reply:
done